www/css/text-bubbles.css

```diff
@@ -0,0 +1,294 @@
+  display: block;
+  width: auto;
+  max-width: 100%;
+  padding: 4px 5px;
+  border: 1px solid var(--netizen-hint-shadow);
+  background-color: var(--netizen-hint-shadow);
+  color: var(--netizen-meta);
+  font-size: 0.9em;
+}
+.text-bubble > section input::placeholder {
+  color: var(--netizen-meta);
+  opacity: 0.8;
+}
+.text-bubble > section input:focus {
+  outline: 1px solid var(--fg-color);
+  outline-offset: 2px;
+}
+.text-bubble > section a {
+  position: relative;
+  color: var(--fg-color);
+  text-decoration: none;
+  transition: color .5s ease, border .5s ease;
+
+  /*underline*/
+  border-bottom: 1px solid var(--fg-color);
+  text-shadow: 2px 2px var(--bg-color), 2px -2px var(--bg-color), -2px 2px var(--bg-color), -2px -2px var(--bg-color);
+}
+.text-bubble > section a:hover {
+  color: var(--netizen-match-color);
+  border-bottom: 1px solid var(--netizen-match-color);
+}
+.text-bubble > section a:active {
+  color: var(--netizen-attribute);
+}
+.text-bubble > section a:active:after {
+  background-color: var(--netizen-attribute);
+}
+
+
+/* styles for all possible text bubble h1s/headers */
+.text-bubble > section h1 {
+  display: inline-block;
+  position: relative;
+  width: auto;
+  margin-bottom: 12px;
+  color: var(--fg-color);
+  font-size: 20px;
+  line-height: 1.2em;
+  font-weight: 400;
+  transform: skew(0);
+  transition: color .2s ease;
+  z-index: 1;
+}
+
+.text-bubble > section h1 a {
+  display: inline-block;
+  position: relative;
+  width: auto;
+  color: var(--fg-color);
+  text-decoration: none;
+  transition: color .2s ease;
+
+  /* resets */
+  background: transparent;
+  border-bottom: none;
+  padding-bottom: 0;
+  text-shadow: none;
+}
+
+.text-bubble > section h1 a:after {
+  display: none;
+}
+
+.text-bubble > section h1 a:hover {
+  /* resets */
+  color: inherit;
+  border-bottom: none;
+  background-image: none; 
+}
+
+.text-bubble > section h1 code {
+  display: inline-block;
+  position: relative;
+  width: auto;
+  color: var(--fg-color);
+  text-decoration: none;
+  transition: color .2s ease;
+  /* resets */
+  top: auto;
+  padding: 0;
+  border-radius: 0;
+  border: none;
+  background-color: transparent;
+  font-size: 20px;
+  line-height: 1.2em;
+  cursor: pointer;
+}
+
+.text-bubble > section h1:before {
+  content: '';
+  position: absolute;
+  display: block;
+  top: 1px;
+  left: -2px;
+
+  width: 100%;
+  height: 95%;
+
+  padding: 0px 2px;
+
+  background-color: var(--netizen-hint-shadow);
+
+  z-index: -1;
+
+  transform: scaleX(0);
+  transform-origin: right;
+  transition: transform .5s cubic-bezier(0.165, 0.84, 0.44, 1);
+}
+
+.text-bubble > section h1:after {
+  content: '';
+  position: absolute;
+  display: block;
+  top: 1px;
+  right: -4px;
+
+  width: 2px;
+  height: 95%;
+
+  background-color: var(--fg-color);
+  background-color: #ffffff;
+
+  animation: blink 1.5s infinite steps(1);
+  transition: right .5s cubic-bezier(0.165, 0.84, 0.44, 1);
+}
+
+@keyframes blink {
```

Comment:
will do!

Reply:
ok weird.... is "blink" some sort of keyword in css? o__O when i change the name of the animation to literally anything else, the animation itself breaks.. lol i'm not sure if i'm just doing something really obvious wrong

Reply:
weird... just tried it out myself, same issue... bizarre. i have noticed weird behavior with CSS animation names in the past though. one time i had an animation that wasn't working && i was able to fix it just by changing the name... wish i could remember what the name was, but i do remember thinking the same thing, "could this be a keyword?" ...i wish i would have looked it up (but i never did).

what does the `steps(1)` do? i noticed that when i remove it, it does show up (but not w/the same effect, it fades instead of ...well, blinking?)

Reply:
...also, if this is gonna become a whole thing, then let's just forge it, not worth wasting too much time on it (we'll just keep in mind that as far as animation names go "blink" is taken) ...though, the weird behavior does give me pause... like, i hope it's not one of those things that reappears as some other weird issue down the line O_o

Reply:
ooo maybe steps() isnt entirely supported?! its a timing function that like, renders each change in an animation as a... step? instead of like, a smooth tween or transition. not sure how to describe it, but its what's causing the 0 -> 1 opacity to "blink" instead of fade in and out.

anyway, yeah, i tried what you did and changed `steps(1)` to `linear` and had to change up the timing of the animation a little too, and its working fine now with a new name. so weird bc i've definitely used steps() before... my brain feels like ???

Reply:
oh! dang, that's too bad `steps()` sounds super useful! everytime i've wanted a "hard jump" like that i've also done what u did now `25% {}, 26% {}`... but that's always felt hacky. still find it strange that `steps()` works if/when the animation has the name "blink" but not something else... that's bizarre